CHANGELOG.md

```diff
@@ -5,6 +5,12 @@
 A breaking change should be clearly marked in this log.
 
 
+## v1.0.0-beta.4
```

Comment:
should we mention in here that this sdk is deprecated and soon eol and users should transition to js-stellar-sdk?